core/src/main/java/nl/nn/adapterframework/extensions/javascript/Nashorn.java

```diff
@@ -0,0 +1,97 @@
+package nl.nn.adapterframework.extensions.javascript;
+
+import jdk.nashorn.api.scripting.NashornScriptEngine;
```

Comment:
I think you can just use ScriptEngine here. There is nothing special/specific happening in the `NashornScriptEngine` class afaik.